bom/application/pom.xml

```diff
@@ -44,7 +44,7 @@
         <smallrye-health.version>3.3.0</smallrye-health.version>
         <smallrye-metrics.version>3.0.5</smallrye-metrics.version>
         <smallrye-open-api.version>2.3.0</smallrye-open-api.version>
-        <smallrye-graphql.version>1.8.0</smallrye-graphql.version>
+        <smallrye-graphql.version>2.0.0-SNAPSHOT</smallrye-graphql.version>
```

Comment:
Just a note that we (obviously) need a proper a release before merging this so that it doesn't get merged by mistake.

Reply:
Yeah, @jmartisk will backport to 1.8.x and create a release. I will update this PR when that is done.